packages/tf2-dm/Dockerfile

```diff
@@ -8,7 +8,7 @@ RUN rm "${SERVER_DIR}/tf/addons/sourcemod/plugins/"{funcommands,funvotes}.smx
 RUN \
   # download plugins
   SOAP_DM_FILE_NAME="soap-dm.zip" \
-  && wget -nv "https://github.com/spiretf/SOAP-TF2DM/archive/master.zip" -O "${SOAP_DM_FILE_NAME}" \
+  && wget -nv "https://github.com/sapphonie/SOAP-TF2DM/archive/master.zip" -O "${SOAP_DM_FILE_NAME}" \
```

Comment:
The package should be downloaded from the releases page, so we can have a proper versioning of the plugin. See https://github.com/melkortf/tf2-servers/pull/17/files

Reply:
Also, the MD5 checksums won't match, the build fill fail.